lib/mutant/meta.rb

```diff
@@ -14,9 +14,9 @@ class Example
       # Add example
       #
       # @return [undefined]
-      def self.add(&block)
+      def self.add(type = nil, &block)
```

Comment:
Should this be `type = Undefined` to distinguish between the case where an explicit `nil` was passed in vs. the case where no argument was passed in?